src/main/java/ui/components/issuepicker/IssuePickerDialog.java

```diff
@@ -117,7 +117,7 @@ private void populateSelectedIssues(Optional<TurboIssue> selectedIssue) {
         selectedIssues.getChildren().clear();
         FlowPane selectedIssueCards = getSelectedIssuesPane(selectedIssue);
         if (!selectedIssueCards.getChildren().isEmpty()) {
-            selectedIssues.getChildren().addAll(createRepoTitle(models.getDefaultRepo()), selectedIssueCards);
+            selectedIssues.getChildren().addAll(createRepoTitle(selectedIssue.get().getRepoId()), selectedIssueCards);
```

Comment:
If the selected issue is guaranteed to be here at this point, the method should take it directly instead of an optional.